cpa/tester/policies/helpers/contexts/policy.rego

```diff
@@ -0,0 +1,17 @@
+package org
+
+import data.circleci.config
+
+policy_name["context_lists"]
+
+enable_rule["context_allowlist"]
+
+context_allowlist := config.context_allowlist_by_project_id(data.meta.project_list, data.meta.allowlist)
```

Comment:
As we wish to navigate away from project_ids and towards project_names, I think these helpers should also be defined upon names instead.

Reply:
There seems to be a lot problems with project_slugs right now.

Let's release the project_id version and then we can make a ticket to release the project_slug version once the slug version is properly released and stabilized from builds.